test/unit/sanitizer/_geo_reverse.js

```diff
@@ -61,27 +61,88 @@ module.exports.tests.warning_situations = (test, common) => {
 };
 
 module.exports.tests.success_conditions = (test, common) => {
+  // note: this behaviour is historic and should probably be removed.
+  // it's possible to add non-numeric tokens to the boundary.circle.radius
+  // value and they are simply stripped out by parseFloat() with warning.
+  test('boundary.circle.radius must be a positive number', (t) => {
+    const raw = {
+      'point.lat': '12.121212',
+      'point.lon': '21.212121',
+      'boundary.circle.radius': '1km' // note the 'km'
```

Comment:
this is weird, I'd like to change it, but in another PR.
right now you can specify `5km` or `5m` and in both cases the unit designation is stripped and results in `5`, which is `km`.